TechtonicaModLoader/MVVM/Mod/ModViewModel.cs

```diff
@@ -1,18 +1,16 @@
 ﻿using CommunityToolkit.Mvvm.ComponentModel;
 using CommunityToolkit.Mvvm.Input;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using TechtonicaModLoader.Models;
```

Comment:
You'll see that I created two new folders and new namespaces. This is handy for catching potential issues.

Reply:
Understood and in a different context I would wholly support this.  But Views, Models and ViewModels are not always 1-to-1-to-1 like this.  

My goal, though, was to have each layer in its own namespace to help make it more obvious when, from a MVVM architecture perspective, you can and cannot use something from the file you are working on.   

Folder layout can be whatever but this was both taking advantage of VS auto-namespacing by folder and making the separation super-clear when it wasn't in the name.